src/pages/settings/Security/TwoFactorAuth/TwoFactorAuthPropTypes.tsx

```diff
@@ -0,0 +1,24 @@
+import type { OnyxEntry } from 'react-native-onyx';
```

Comment:
move types to files where there are used , we use only seperate type files where there are platform specific files https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md#file-organization